apps/mobile/src/features/approver/clarity-value.section.tsx

```diff
@@ -0,0 +1,70 @@
+import { useTheme } from '@shopify/restyle';
+import { ClarityType, ClarityValue, cvToString } from '@stacks/transactions';
+
+import { Box, Text, Theme } from '@leather.io/ui/native';
+
+function wrapText(val: string) {
+  return <Text>{val}</Text>;
+}
+
+export function ClarityValueListDisplayer({
+  val,
+  level = 0,
+}: {
+  val: ClarityValue;
+  level?: number;
+}) {
+  const theme = useTheme<Theme>();
+  const encoding = 'tryAscii';
+
+  switch (val.type) {
+    case ClarityType.BoolTrue:
+      return wrapText('true');
```

Comment:
Could we just use `<Text>` here? I'm not sure if `wrapText` adds much benefit

Reply:
Yeah, have no idea where `wrapText` came from ...even predates me, haha.

Reply:
Thinking on what Fara said and maybe we could share this with the extension, using `wrapText` to our advantage. 

We could pass that in as a callback that uses either `Box` / `Text` depending on platform. Then we could share all of it if we also pass in a component for `ClarityType.Tuple`